From 563dcc0f92a0712c29acbc2669968b3d0a1edb77 Mon Sep 17 00:00:00 2001
From: He Zhe <zhe.he@windriver.com>
Date: Tue, 19 Jun 2018 00:58:18 -0700
Subject: [PATCH] yaffs: Fix build failure by removing unused members of
 erase_info

A few members of struct erase_info have been removed and no longer
used in mainline by the following commits. This patches removes
related useless initialization.

884cfd9023ce6afe8bcf181ec988d8516eb32bf0
"mtd: Stop assuming mtd_erase() is asynchronous"

Signed-off-by: He Zhe <zhe.he@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
Signed-off-by: Yue Tao <Yue.Tao@windriver.com>
---
 fs/yaffs2/yaffs_mtdif.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/fs/yaffs2/yaffs_mtdif.c b/fs/yaffs2/yaffs_mtdif.c
index fa4b99834570..561784c15196 100644
--- a/fs/yaffs2/yaffs_mtdif.c
+++ b/fs/yaffs2/yaffs_mtdif.c
@@ -59,8 +59,6 @@ int nandmtd_erase_block(struct yaffs_dev *dev, int block_no)
 	ei.len = dev->param.total_bytes_per_chunk * dev->param.chunks_per_block;
 	ei.time = 1000;
 	ei.retries = 2;
-	ei.callback = NULL;
-	ei.priv = (u_long) dev;
 
 	retval = mtd_erase(mtd, &ei);
 
@@ -188,8 +186,6 @@ static 	int yaffs_mtd_erase(struct yaffs_dev *dev, int block_no)
 	ei.len = block_size;
 	ei.time = 1000;
 	ei.retries = 2;
-	ei.callback = NULL;
-	ei.priv = (u_long) dev;
 
 	retval = mtd_erase(mtd, &ei);
 
-- 
2.17.1

